data/drules_proto_vehicle_clear.txt

```diff
@@ -6904,6 +6906,7 @@ cont {
       secondary {
         height: 8
         color: 6381914
+        offset_y: 1
```

Comment:
Why is this offset needed? Which problem does it solve? Is it possible to fix the issue without introducing the offset?

Reply:
ATM its used to set text's anchor only (below icon, above, to the left...). The actual offset value is not needed. Maybe setting anchor is not needed also - its set to below icon everywhere (if there is an icon) and only housenumbers position is hardcoded (its either center or above).
So. Offsets are not needed for sure. And I'd prefer to get rid of setting anchor positions (simplify and hardcode).

Reply:
Apple maps uses different anchors:
![telegram-cloud-photo-size-2-5282894737579297686-y](https://github.com/organicmaps/organicmaps/assets/170263/9b787cfa-b3c5-402b-bd2c-95f03f933b42)

Reply:
and anchors are not fixed to poi types (like ours), but look dynamic to avoid conflicts with other overlays, which make more sense

Reply:
Yes, so we can remove anchors now from styles, and also make them dynamic )